modules/c/examples/code_snippets/main.c

```diff
@@ -16,14 +17,92 @@ void usleep(unsigned int us) {
 static CBLDatabase* kDatabase;
 static CBLReplicator* kReplicator;
 
+
+//  BEGIN lower-level function declarations
+
+//  DOCS NOTE --
+//  These functions are referred to in subsequent code samples.
+//  Their tags will ensure they are shown alongide the usage examples.
+//  Functions used in more than one place may hve multiple tags.
+
+// tag::p2p-act-rep-func[]
+// tag::p2p-act-rep-add-change-listener[]
```

Comment:
Won't this show the tag comment inside of p2p-act-rep-add-change-listener ?

Reply:
No there it filters out 'tags'